cockatrice/src/dlg_edit_avatar.cpp

```diff
@@ -75,6 +77,13 @@ QByteArray DlgEditAvatar::getImage()
     QByteArray ba;
     QBuffer buffer(&ba);
     buffer.open(QIODevice::WriteOnly);
-    image.save(&buffer, "JPG");
-    return ba;
+    for (;;) {
```

Comment:
any reason you do `for (;;)` instead of `while (true)`?

Reply:
I just prefer it, I think it looks nice